### PR TITLE
new try: request manager

### PR DESCRIPTION
**Pull request by AaronClaros**

### CODE REVIEW

File: e2e/specs/create.project.spec.js [hunk 1]
```diff
@@ -1,47 +1,31 @@
...
-        expect(`${projectData3.name} - Pivotal Tracker`).to.not.equal(project.getProjectName());
-    });
-    it('should create a new private project with a new account', () => {
-        project = dashboard.createProject(projectData4);
-        expect(`${projectData4.name} - Pivotal Tracker`).to.equal(project.getProjectName());
+
+    after(() =>{
+        let projectID;
+        let response = browser.call(() => {return APIrequest.GetRequest('projects');});
+        Object.values(response.data).map((project) => {
```
Comment (carledriss) [inline e2e/specs/create.project.spec.js]: BLOCKER Add a "return" statement to this callback. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3796)

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue
* ![BLOCKER][BLOCKER] 1 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'

File: e2e/specs/create.project.spec.js [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
-        project = dashboard.createProject(projectData3);
-        expect(`${projectData3.name} - Pivotal Tracker`).to.not.equal(project.getProjectName());
-    });
-    it('should create a new private project with a new account', () => {
-        project = dashboard.createProject(projectData4);
-        expect(`${projectData4.name} - Pivotal Tracker`).to.equal(project.getProjectName());
+
+    after(() =>{
+        let response = browser.call(() => {return APIrequest.GetRequest('projects');});
+        let projectID = Object.values(response.data).map((project) => {
```
Comment (carledriss) [inline e2e/specs/create.project.spec.js:23]: MAJOR Refactor this function to use "return" consistently. [rule](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3801)

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by anghelitalo — COMMENTED** [review]
done

**Review by anghelitalo — APPROVED** [review]

**Pull request merged**